File: C#/Урок 038 - while/Задача 03.cs
class Program
{
    static void Main()
    {
        System.Console.Write("Enter a positive number: ");
        int userNumber = System.Convert.ToInt32(System.Console.ReadLine());
        while (userNumber <= 0)
        {
            System.Console.WriteLine("Your number is not positive. Try again.");
            userNumber = System.Convert.ToInt32(System.Console.ReadLine());
        }

        int number = 0;
        while (number <= userNumber)
        {
            System.Console.Write(number + " ");
            number++;
        }
    }
}

File: C++/Урок 028 - if/Задача 03.cpp
#include <iostream>

int main()
{
    std::cout << "Enter a number: ";
    int number;
    std::cin >> number;

    if (number > 0)
    {
        std::cout << "Your number is positive\n";
    }
    else if (number == 0)
    {
        std::cout << "Your number is zero\n";
    }
    else
    {
        std::cout << "Your number is negative\n";
    }

	return 0;
}